### PR TITLE
chore: enable progress tracker for all stack set operations

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/deploy/cloudformation/app.go [hunk 1]
```diff
@@ -279,7 +284,7 @@ func (cf CloudFormation) addWorkloadToApp(app *config.Application, wlName string
 		Accounts: previouslyDeployedConfig.Accounts,
 		App:      appConfig.Name,
 	}
-	if err := cf.deployAppConfig(appConfig, &newDeploymentConfig); err != nil {
+	if err := cf.deployAppConfig(appConfig, &newDeploymentConfig, shouldAddNewWl); err != nil {
```
Comment (dannyrandall) [inline internal/pkg/deploy/cloudformation/app.go:287]: isn't this always true?

(i'm not sure i'd even want switch it to just `true`, but just )

Comment (dannyrandall) [reply internal/pkg/deploy/cloudformation/app.go:287]: yeah i like it the way you have it 😊

**Review by dannyrandall — APPROVED** [review]
lgtm!

**Review by iamhopaul123 — APPROVED** [review]
:shipit:

**Pull request merged**